src/game/AI/ScriptDevAI/scripts/outland/shattrath_city.cpp

```diff
@@ -28,8 +28,13 @@ npc_khadgars_servant
 npc_salsalabim
 EndContentData */
 
+#include "AI/BaseAI/AIDefines.h"
+#include "AI/ScriptDevAI/ScriptDevAIMgr.h"
 #include "AI/ScriptDevAI/include/sc_common.h"
 #include "AI/ScriptDevAI/base/escort_ai.h"
+#include "AI/ScriptDevAI/include/sc_creature.h"
+#include "Entities/Creature.h"
+#include "Entities/Unit.h"
```

Comment:
these are as always added automatically by my IDE, I'll remove them on the final step for cleanup